src/yubikey.rs

```diff
@@ -633,53 +625,6 @@ impl YubiKey {
         txn.save_object(object_id, indata)
     }
 
-    /// Get an auth challenge.
-    #[cfg(feature = "untested")]
-    pub fn get_auth_challenge(&mut self) -> Result<[u8; 8]> {
```

Comment:
I guess this PR is anyway a breaking API change, so maybe it doesn't matter, but: is there a reason not to preserve the API w.r.t. `get_auth_challenge` and `verify_auth_response`? I suppose they'd have to take a generic argument...

Reply:
Ah, very sorry! I missed the commit message regarding this.